landing-page.html

```diff
@@ -54,12 +54,21 @@ <h2>Latest Videos</h2>
   <h2>Support The Coding Train</h2>
   <div class="support">
     {% capture patreonLogo %}{% include 4-vector-graphics/patreon.svg %}{% endcapture %}
-    {% include 3-modules/hero-item.html image=patreonLogo title='Consider supporting?' content='Support The Coding Train on Patreon and get some awesome rewards!' url=site.links.patreon cta='Become a Patron' class='patreon' %}
+    {% include 3-modules/hero-item.html image=patreonLogo title='Consider supporting?' content='Join the Coding Train
+    community with a YouTube membership or Patreon subscription!'
```

Comment:
I'll test if you can have a link here but it should work by just including HTML... 🤔

Reply:
Hah, so when I added the HTML links this is what happened. I guess we need a different style/class for inline?

<img width="507" alt="screen shot 2018-10-21 at 3 04 14 pm" src="https://user-images.githubusercontent.com/191758/47271077-b2da2a00-d542-11e8-8887-fe107e959014.png">

Reply:
Going to take the links out for now so this can be merged and we can sort out as a separate issue?

Reply:
Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Reply:
Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Reply:
Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Reply:
Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Reply:
Oh... That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Reply:
Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Reply:
Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Reply:
Oh... 🤔 I'll sort this out in a separate PR so we can merge this one.

Reply:
Is this visible?

Reply:
weird, your comment came in 6 times!